### PR TITLE
Add python3-enchant to Mariner

### PR DESCRIPTION
**Pull request by henryli001**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
- There's customer request to enable python3-enchant in Mariner for their workloads

###### Does this affect the toolchain?  

NO

###### Associated issues  

- https://microsoft.visualstudio.com/OS/_workitems/edit/43041721/

###### Test Methodology

- Buddy Build

### CODE REVIEW

File: SPECS-EXTENDED/python-enchant/python-enchant.spec [hunk 1]
```diff
@@ -0,0 +1,311 @@
+%global srcname enchant
+
+Name:           python-enchant
+Version:        3.2.2
+Release:        7%{?dist}
+Summary:        Python bindings for Enchant spellchecking library
+License:        LGPLv2+
```
Comment (mfrw) [inline SPECS-EXTENDED/python-enchant/python-enchant.spec:7]:
```suggestion
License:        LGPL-2.0-or-later
```

File: SPECS-EXTENDED/python-enchant/python-enchant.spec [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            http://packages.python.org/pyenchant/
```
Comment (mfrw) [inline SPECS-EXTENDED/python-enchant/python-enchant.spec:10]: This URL throws a 404 :(
can we put the github location ? for both url & source ?

File: SPECS-EXTENDED/python-enchant/python-enchant.spec [hunk 3]
```diff
@@ -0,0 +1,311 @@
+%global srcname enchant
+
+Name:           python-enchant
+Version:        3.2.2
+Release:        7%{?dist}
+Summary:        Python bindings for Enchant spellchecking library
```
Comment (mfrw) [inline SPECS-EXTENDED/python-enchant/python-enchant.spec:6]: Maybe we should put summary as the first, as per our convention :)

Comment (henryli001) [reply SPECS-EXTENDED/python-enchant/python-enchant.spec]: Fixed

**Review by mfrw — APPROVED** [review]

**Pull request merged**